types/fibers/index.d.ts

```diff
@@ -1,23 +1,91 @@
-// Type definitions for fibers
+// Type definitions for node-fibers 3.1
```

Comment:
It looks like the lint rules for this package are disabled. Could you at least remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!

Reply:
I believe I have no enabled the linting